packages/common/src/TransactionUtils.ts

```diff
@@ -160,3 +160,60 @@ export const blockUntilBlockMined = async (web3: Web3, blockerBlockNumber: numbe
     await new Promise((r) => setTimeout(r, delay));
   }
 };
+export async function findBlockNumberAtTimestamp(
```

Comment:
this function was taken from here: https://github.com/UMAprotocol/protocol/blob/582464658c17e7ba9d18c357f61e0f2126543f06/packages/affiliates/liquidity-mining/FindBlockAtTimeStamp.js#L41 and modified to work with ts